add-chain/main.go

```diff
@@ -25,24 +25,28 @@ var (
 	ChainTypeFlag = &cli.StringFlag{
 		Name:     "chain-type",
 		Value:    "frontier",
+		EnvVars:  []string{"CHAIN_TYPE"},
```

Comment:
Suggestion: add a envVar helper function, which prefixes the env-var name with `SR_ADD_CHAIN_` (or something like that) to namespace the env vars. We do this with all the op-stack services in the monorepo, to avoid env-var conflicts.

Reply:
Good callout, I'd like to vote for `SCR_` prefix since I think folks would recognise this a bit more readily.